.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
refactor (blocking): Storybook fails to build because the webpack common js loader is unable to parse this file. Adding a `.json` extension will fix the issue.

Reply:
Sorry. I did not double check it. I'll fix it. But weirdly story book was working in dev mode.